use-cases/inferencing/serving-with-vllm/README.md

```diff
@@ -0,0 +1,374 @@
+# Distributed Inferencing on vLLM
```

Comment:
I assume this README will change when we "onboard" it to use the playground, but it would be good if we could get it into a similar format/style as the other guides. I tend to not use numbered list as they get messy, unless you are really using them to enumerate something.